addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -2002,57 +2002,6 @@ test("Chatter notification in messaging menu should open the form view even when
     });
 });
 
-test("Chats input should wait until the previous RPC is done before starting a new one", async () => {
```

Comment:
I've not checked in details if it was there but don't we want a similar test and feature for ctrl + k?

Reply:
ChannelSelector was code in `mail`, so this test encited us to implement with performance in mind.

Now this aspect is managed by the command palette. It as a `async provide()` method that we `await` in it during the whole process of fetching and showing results. I suppose the implementation of command palette already does this and there's likely already some test coverage in `web` for that.

Again, no extra effort needed to maintain a test that is no longer needed (component is removed), and command palette looks already good with that aspect.

Reply:
Except it is not handled in command palette at all. There are as many RPC started in parallel as changes made on the input.

It never waits for the previous one to return, nor cancel it, guarantee they arrive in the correct order, or anything at all. It's just spamming the server and hoping for the best, precisely want we wanted to avoid.

Reply:
By slowing the network enough I could easily be in a situation with 6 RPC waiting in parallel.